### DISCUSSION TITLE
MacOS release

### DISCUSSION THREAD

**Discussion by jakubjanousek** [motor-admin/motor-admin#77] [General]
Hi Pete,
I tried downloading the latest MacOS binary, but saw it's still on v0.3.16 (at least it says so on the bottom of the screen). Is there any plan with releasing 0.4.5?

Thanks a lot,
Jakub

**Comment by omohokcoj** [comment]
@jakubjanousek unfortunately only docker images are available starting from 0.4.0
https://github.com/motor-admin/motor-admin/issues/76#issuecomment-1407373537

**Reply by jakubjanousek** [reply]
@omohokcoj that is perfectly fine, just didn't see the discussion. Thanks for info
